### DISCUSSION TITLE
Notice of/ community approval, of me copying SDRPlusPlus band plans for use in my SDR# repository?

### DISCUSSION THREAD

**Discussion by Arrin-KN1E** [AlexandreRouma/SDRPlusPlus#836] [General]
Hello! I wanted to say I know it is an open-source project, so borrowing work is expected. However, I did not want it to appear as though im stealing the work of the project, especially as it grows. I maintain https://github.com/Arrin-KN1E/SDRSharp-Band-Plans and want to make it more useful by adapting the band plan work done here and into the future. 

I just wanted to notify the SDRPlusPlus community of my actions, and make sure it is okay. I am working on a large update, and this would work perfectly with it!

**Comment by AlexandreRouma** [comment]
Each band plan is created and maintained by different people so I cannot approve it for everyone. You must ask for permission to the people who contributed these bandplans, their contact details are usually listed in each bandplan file.
I'm fine with mine (`Ryzerth`) being used, but see the rest of this comment for details.

This repository is licensed under GPLv3. This means that reuse of the files within this codebase are subject to the terms of the license. This includes, but is not limited to, proper attribution to the project and contributors.

**Comment by Arrin-KN1E** [comment]
@AlexandreRouma Thanks for the reply. I knew I forgot something when writing my post... I will absolutely attribute each file to its respective author! I only plan on adapting it to XML.
I'll contact them for permission before starting the work! 
Thanks!
